projects/systemd/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER jonathan@titanous.com
+RUN echo 'deb-src http://archive.ubuntu.com/ubuntu/ xenial main' >> /etc/apt/sources.list &&\
+    echo 'deb-src http://archive.ubuntu.com/ubuntu/ xenial-updates main' >> /etc/apt/sources.list &&\
+    echo 'deb-src http://security.ubuntu.com/ubuntu/ xenial-security main' >> /etc/apt/sources.list &&\
+    echo 'deb http://archive.ubuntu.com/ubuntu/ artful main' >> /etc/apt/sources.list &&\
```

Comment:
why are these new versions needed?

Reply:
systemd requires a newer version of libmount than xenial has, this was the easiest way I found to accomplish that. Still need to figure out what to do about all the shared library dependencies going forward, but I didn’t want to block on that.

Reply:
I think [`ppa:pitti/systemd-semaphore`](http://ppa.launchpad.net/pitti/systemd-semaphore/ubuntu/dists/) should be used instead. @martinpitt backported all the required libraries, so that such cross-release additions wouldn't be necessary.

Reply:
Perfect, thanks!